src/types.tsx

```diff
@@ -26,5 +26,5 @@ export interface AllQuestionFormat{
   //external_services:QuestionFormat;
   //survey:QuestionFormat;
 
-export const QuestionContext = React.createContext<AllQuestionFormat | {} >({})
+export const QuestionContext = React.createContext<any>({})
```

Comment:
We can try to refactor our interfaces to help alleviate the typing errors we are getting.